content/influxdb/clustered/admin/licensing.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Manage your InfluxDB Clustered license
+description: >
+  Install and manage your InfluxDB Clustered license to authorize the use of
+  the InfluxDB Clustered software.
+menu:
+  influxdb_clustered:
+    parent: Administer InfluxDB Clustered
+    name: Manage your license
+weight: 101
+influxdb/clustered/tags: [licensing]
+related:
+  - /influxdb/clustered/install/licensing/
+  - /influxdb/clustered/admin/upgrade/
+---
+
+Install and manage your InfluxDB Clustered license to authorize the use of
+the InfluxDB Clustered software.
+
+- [Install your InfluxDB license](#install-your-influxdb-license)
+- [Recover from a license misconfiguration](#recover-from-a-license-misconfiguration)
+- [Renew your license](#renew-your-license)
+- [License enforcement](#license-enforcement)
+  - [A valid license is required](#a-valid-license-is-required)
+  - [Periodic license checks](#periodic-license-checks)
+  - [License grace periods](#license-grace-periods)
+    - [License expiry logs](#license-expiry-logs)
+    - [Query brownout](#query-brownout)
+
+{{% note %}}
+#### License enforcement is currently an opt-in feature
+
+In currently available versions of InfluxDB Clustered, license enforcement is an
+opt-in feature that allows InfluxData to introduce license enforcement to
+customers, and allows customers to deactivate the feature if issues arise.
+In the future, all releases of InfluxDB Clustered will require an active license
+to use the product.
+
+To opt into license enforcement, include the `useLicensedBinaries` feature flag
+in your `AppInstance` resource _([See the example below](#enable-feature-flag))_.
+To deactivate license enforcement, remove the `useLicensedBinaries` feature flag.
+{{% /note %}}
+
+## Install your InfluxDB license
+
+{{% note %}}
+If setting up an InfluxDB Clustered deployment for the first time, first
+[set up the prerequisites](/influxdb/clustered/install/licensing/) and
+[configure your cluster](/influxdb/clustered/install/configure-cluster/).
+You will be able to install your license after the `influxdb` namespace is
+created and prepared.
```

Comment:
```suggestion
[configure your cluster](/influxdb/clustered/install/configure-cluster/).
The configuration process creates a default `influxdb` namespace in your cluster.
After the `influxdb` namespace is
created and prepared, you will be able to install your license.
```

Reply:
Sorry if this has been discussed, but what if they use a different namespace name (see https://docs.influxdata.com/influxdb/clustered/install/configure-cluster/#create-a-namespace-for-influxdb)? Does it still work?

Reply:
They can use whatever namespace name they want. I'm going to make this generic. The install docs use `influxdb` as the namespace name.